empress/recon_vis/recon_viewer.py

```diff
@@ -96,83 +222,358 @@ def render_parasite_helper(fig, node, recon, host_lookup, show_internal_labels,
+    mid_xy = (node_xy[0], current_xy[1])
+
+    fig.line(node_xy, mid_xy, PARASITE_EDGE_COLOR)
+    fig.line(mid_xy, current_xy, PARASITE_EDGE_COLOR)
+
+def event_color_shape(event):
+    """
+    Gives the color and shape for drawing event, depending on event type
+    :param event: Event object
+    :return A tuple with the color and shape of an event
+    """
     if event.event_type is EventType.TIPTIP:
-        return LEAF_NODE_COLOR
+        return LEAF_NODE_COLOR, LEAF_NODE_SHAPE
     if event.event_type is EventType.COSPECIATION:
-        return COSPECIATION_NODE_COLOR
+        return COSPECIATION_NODE_COLOR, COSPECIATION_NODE_SHAPE
     if event.event_type is EventType.DUPLICATION:
-        return DUPLICATION_NODE_COLOR
+        return DUPLICATION_NODE_COLOR, DUPLICATION_NODE_SHAPE
     if event.event_type is EventType.TRANSFER:
-        return TRANSFER_NODE_COLOR
+        return TRANSFER_NODE_COLOR, TRANSFER_NODE_SHAPE
     return plot_tools.BLACK
 
-
-def set_host_node_layout(host_tree: tree.Tree):
+def set_host_node_layout(host_tree):
     """
     Sets the logicalRow and logicalCol values of each Node in host_tree.
     Assumes that each host Node has its order set already and this function
     uses those values and structure of the tree to set the logicalRow and logicalCol
     :param host_tree:  A Tree object representing the host tree
     :return None
     """
-    # sets logical row values for leaves in the order they appear in the list of host tree leaves
+    #sets logical row values for leaves in the order they appear in the list of host tree leaves
     logical_row_counter = 0
-    for leaf in host_tree.leaf_list():
+    for leaf in host_tree.leaf_list:
         leaf.layout.row = logical_row_counter
-        leaf.layout.x = leaf.layout.col  # This can be scaled if desired
-        leaf.layout.y = leaf.layout.row  # This can be scaled if desired
+        leaf.layout.x = leaf.layout.col           # This can be scaled if desired
+        leaf.layout.y = leaf.layout.row           # This can be scaled if desired
         logical_row_counter += 1
-    # helper function to assign row values, postorder traversal
+    #helper function to assign row values, postorder traversal
     set_internal_host_nodes(host_tree.root_node)
 
-
-def set_internal_host_nodes(node: tree.Node):
-    """ Helper function for set_host_node_layout. """
-    if node.is_leaf():
+def set_internal_host_nodes(node):
+    """
+    Helper function for set_host_node_layout
+    :param node: Node object
+    """
+    if node.is_leaf:
         return
     set_internal_host_nodes(node.left_node)
     set_internal_host_nodes(node.right_node)
-    node.layout.row = (node.left_node.layout.row + node.right_node.layout.row) / 2
-    node.layout.x = node.layout.col  # This can be scaled if desired
-    node.layout.y = node.layout.row  # This can be scaled if desired
+    node.layout.row = (node.left_node.layout.row + node.right_node.layout.row)/2
```

Comment:
Let's use two spaces for comments.
```python
node.layout.x = node.layout.col         # This can be scaled if desired
# change to
node.layout.x = node.layout.col  # This can be scaled if desired
```

See:
https://google.github.io/styleguide/pyguide.html#36-whitespace